### PR TITLE
Profiles: Misc fixes

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-3250
Fixes RNBW-3251

## What changed (plus any additional context for devs)

This PR fixes various minor issues we have been seeing around avatar upload & accent colors.

### CODE REVIEW

File: src/screens/ENSConfirmRegisterSheet.js [hunk 1]
```diff
@@ -327,13 +322,6 @@ export default function ENSConfirmRegisterSheet() {
     getReverseRecord();
   }, [accountAddress]);
 
-  useEffect(() => {
```
Comment (jxom) [inline src/screens/ENSConfirmRegisterSheet.js:330]: Don't think we needed this as we are already doing this with `ENSAssignRecordsSheet`

File: src/components/expanded-state/unique-token/ENSBriefTokenInfoRow.tsx [hunk 2]
```diff
@@ -32,6 +33,8 @@ export default function ENSBriefTokenInfoRow({
     startRegistration(ensName, REGISTRATION_MODES.RENEW);
     navigate(Routes.ENS_CONFIRM_REGISTER_SHEET, {
       ensName,
+      hideAvatar: true,
```
Comment (estebanmino) [inline src/components/expanded-state/unique-token/ENSBriefTokenInfoRow.tsx:36]: why do we want to hide the avatar?

**Review by estebanmino — APPROVED** [review]
lgtm!

File: src/components/expanded-state/unique-token/ENSBriefTokenInfoRow.tsx (hunk 2 above)
Comment (jxom) [reply src/components/expanded-state/unique-token/ENSBriefTokenInfoRow.tsx]: we probs don't, i reverted!

**Pull request merged**